modules/call-dispatch/src/lib.rs

```diff
@@ -183,7 +185,17 @@ impl<T: Config<I>, I: Instance> MessageDispatch<T::MessageId> for Module<T, I> {
 		message.weight
 	}
 
-	fn dispatch(bridge: InstanceId, id: T::MessageId, message: Self::Message) {
+	fn dispatch(bridge: InstanceId, id: T::MessageId, message: Result<Self::Message, ()>) {
```

Comment:
Since we're not actually using the `Err` type of the `Result` we might as well just make this an `Option<Self::Message>`